ui/src/app/components/rollout/rollout.tsx

```diff
@@ -305,9 +307,14 @@ const Step = (props: {step: GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1
     return (
         <React.Fragment>
             <EffectDiv className={`steps__step ${props.complete ? 'steps__step--complete' : ''} ${props.current ? 'steps__step--current' : ''}`}>
-                <div className={`steps__step-title ${props.step.experiment ? 'steps__step-title--experiment' : ''}`}>
-                    <i className={`fa ${icon}`} /> {content}
+                <div className={`steps__step-title ${props.step.experiment || (props.step.setCanaryScale && open) ? 'steps__step-title--experiment' : ''}`}>
+                    {icon && <i className={`fa ${icon}`} />} {content}
                     {unit}
+                    {props.step.setCanaryScale && (
```

Comment:
can you test if setcanaryscale is not set and when you render ..is UI works fine or not?

Reply:
UI doesn't show canary scale if setCanaryScale is not set.